tests/func/test_data_cloud.py

```diff
@@ -449,6 +452,25 @@ def _get_url(self):
     def _get_cloud_class(self):
         return RemoteSSH
 
+    def test_symlinks(self):
+        self._setup_cloud()
+
+        remote = self.cloud.get_remote()
+
+        os.remove("bar")
+
+        def urlinfo(path):
+            cwd = os.getcwd()
+            path = posixpath.abspath(os.path.join(cwd, path))
+            netloc = remote.path_info.netloc
+            return URLInfo("ssh://{}{}".format(netloc, path))
```

Comment:
Should not be needed. Won't this work:
```python
from_info = remote.path_info / "foo"
to_info = remote.path_info / "bar"
```

Reply:
@Suor , the `remote` path is different from the one in the current directory:
```python
>>> remote.path_info.path
'/tmp/dvc-test.32712.lguntid7.CUvCFsrBf7cyEA4fndhtkW'

>>> os.getcwd()
'/tmp/dvc-test.32712.8ft94g_9.RfMwGpxMVM6wqkCHScN5bk'
```

If it shouldn't be needed, then we need to address this question:

> Why are we relying on URLInfo to make the connection and not reading from the config file or using the information of the remote?